### ISSUE TITLE
Accessibility: Get in touch button text low contrast when focused

### ISSUE THREAD

**Issue by iamjolly** [18F/18f.gsa.gov#3231]
## Expected Behavior

Homepage hero button text has sufficient contrast with `:focus` and normal states.

## Actual Behavior

Homepage hero button text has low contrast (fails [WCAG 2.0 1.4.3 Contrast Minimum](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)) when it receives `:focus`.

## Steps to reproduce the behavior
- [ ] Open [18f.gsa.gov](https://18f.gsa.gov/) homepage
- [ ] Tab to hero button that says "Get in touch"
- [ ] Note the low contrast text when button has focus.

## This issue is done when:
- [ ] Focus state of button uses higher contrast text (with contrast ratio of > 4.5:1) against the button's background color.

![Screen Shot of low contrast text on focus](https://user-images.githubusercontent.com/1093423/91588215-112d7c00-e915-11ea-9640-8f02487cf830.png)
